src/emscripten-optimizer/optimizer-shared.cpp

```diff
@@ -67,7 +56,7 @@ AsmType detectType(Ref node, AsmData *asmData, bool inVarDef, IString minifiedFr
   switch (node[0]->getCString()[0]) {
     case 'n': {
       if (node[0] == NUM) {
-        if (!isInteger(node[1]->getNumber())) return ASM_DOUBLE;
+        if (!wasm::isInteger(node[1]->getNumber())) return ASM_DOUBLE;
```

Comment:
Do you mean places in binaryen? If so then having it in `emscripten_optimizer/` seems fine. Or do you mean places in other projects?